### PR TITLE
Added Astronomy Major

### PR DESCRIPTION
**Pull request by jason-tung**
### Summary 
Added ASTRO major
https://www.notion.so/courseplan/Add-the-Astronomy-STS-major-1d6c1972ba684cec8fc21d49e70f8b1c

image
image

### Test Plan 
Playtest on staging :~)

### CODE REVIEW

File: src/requirements/data/majors/astro.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      ['PHYS 2214', 'PHYS 2218']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1],
+    slotNames: ['PHYS 1112 or PHYS 1116', 'PHYS 2213 or PHYS 2217', 'PHYS 2214 or PHYS 2218'],
+  },
+  {
+    name: 'Introductory Mathematics Sequence',
+    description: `Three semesters of math including: 
+    MATH 1910, MATH 1120, or MATH 1220, MATH 1920, MATH 2220, or MATH 2240, and MATH 2930, MATH 4710, or ASTRO 3340`,
```
Comment (hahnbeelee) [inline src/requirements/data/majors/astro.ts:22]: Nit - add periods to the ends of all of your descriptions

File: src/requirements/data/majors/astro.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        checker: includesWithSubRequirements(
+          ['ASTRO 4431', 'ASTRO 4432', 'ASTRO 4433'],
+          ['PHYS 4230', 'AEP 4230'],
+          ['PHYS 3316', 'PHYS 3318', 'PHYS 3327', 'PHYS 4443'],
+          ['MATH 2940', 'MATH 2210', 'MATH 2230'],
+          ['MATH ****']
+        ),
+        perSlotMinCount: [2, 1, 4, 1, 2],
+        slotNames: [
+          'Two semesters of Advanced Astrophysics',
```
Comment (hahnbeelee) [inline src/requirements/data/majors/astro.ts:67]: Change this to just 'Advanced Astrophysics' bc it looks verbose in the requirements bar
image

File: src/requirements/data/majors/astro.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          'Thermodynamics',
+          'Four semesters of physics',
```
Comment (hahnbeelee) [inline src/requirements/data/majors/astro.ts:69]: change to 'Physics'

File: src/requirements/data/majors/astro.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+          'Linear Algebra',
+          'Two additional semesters of mathematics',
```
Comment (hahnbeelee) [inline src/requirements/data/majors/astro.ts:71]: Mathematics Course

**Review by hahnbeelee — CHANGES_REQUESTED** [review]
left some comments on some string formatting, but other than that the general logic looks good :)

**Review by tcho6319 — APPROVED** [review]
LGTM! Thanks Jason. If @hahnbeelee's good with your updated changes as well, this PR should be g2g.

**Review by hahnbeelee — APPROVED** [review]
LGTM!

**Pull request merged**